pep-0698.rst

```diff
@@ -418,6 +418,53 @@ hands of project owners, whereas the use of ``@require_explicit_overrides`` in
 libraries would force project owners to use ``@override`` even if they prefer
 not to.
 
+Include the name of the ancestor class being overridden
+-------------------------------------------------------
+
+We considered allowing the caller of ``@override`` to specify a specific
+ancestor class where the overridden method should be defined:
+
+.. code-block:: python
+
+    class Parent0:
+        def foo() -> int:
+            return 1
+
+
+    class Parent1:
+        def bar() -> int:
+            return 1
+
+
+    class Child(Parent0, Parent1):
+        @override(Parent0)  # okay, Parent0 defines foo
+        def foo() -> int:
+            return 2
+
+        @override(Parent0)  # type error, Parent0 does not define bar
+        def bar() -> int:
+            return 2
+
+
+This could be useful for code readability because it makes the override
+structure more explicit for deep inheritance trees. It also might catch bugs by
+prompting developers to check that the implementation of an override still makes
+sense whenever a method being overridden moves from one base class to another.
+
+We decided against it because:
+
+- Supporting this would add complexity to the implementation of both
+  ``@override`` and type checker support for it, so there would need to
+  be considerable value add.
```

Comment:
Shouldn't this be "added value"? I'm not sure I've ever heard "value add" in this way.

Reply:
It's a bit business jargony, means the same as "added value".

* https://en.wiktionary.org/wiki/value-add
* https://www.forbes.com/sites/lizryan/2016/03/01/do-you-speak-corporate-business-jargon-for-humans/

"Added value" may be clearer, or even "benefits".

Reply:
FWIW, I usually hear it with an indefinite article, i.e. "_a_ considerable value add". From an editing perspective, if a change is going to be made, I agree with Hugo that it would be much clearer (especially to a wide, international audience) to just say "benefits".

Reply:
Agreed, this wasn't great wording and "benefits" would be clearer. I can put that change up